### ISSUE TITLE
tota11y: accessibility toolkit

### ISSUE THREAD

**Issue by orioltf** [orioltf/githubListing#340]
accessibility visualization toolkit

tota11y helps visualize how your site performs with assistive technologies. [Check out the announcement blog post](http://engineering.khanacademy.org/posts/tota11y.htm).

The process of testing for accessibility (a11y) is often tedious and confusing. In many cases, developers must have some prior accessibility knowledge in order to make sense of the results.

Instead, tota11y aims to reduce this barrier of entry by helping visualize accessibility violations (and successes), while educating on best practices.

https://khan.github.io/tota11y/
https://github.com/Khan/tota11y
